contrib/forms/ped_fever/save.php

```diff
@@ -15,35 +15,38 @@
 }
 
 if ($_GET["mode"] == "new") {
-    $newid = formSubmit("form_ped_fever", $_POST, $_GET["id"], $userauthorized);
+    $newid = formSubmit("form_ped_fever", $_POST, add_escape_custom($_GET["id"]), $userauthorized);
```

Comment:
Remove the above add_escape_custom()

Reply:
FYI these commits are only going to be focused on SQL Injection (because it's sooooo boring). So I might skip over functions and stuff, but I'll get them on the second pass.

Reply:
TODO for me